Gemfile

```diff
@@ -4,6 +4,7 @@ source "https://rubygems.org"
 gemspec :name => "jekyll"
 
 gem "rake", "~> 12.0"
+gem "require_all"
```

Comment:
What is the role of this gem here..?

Reply:
Shoot. The role of that gem there is to teach me to be more mindful of what files I've changed!

That is an error, and should not have been saved in there.

My understanding is, to update this pull request, I should make the change on my branch, repeat git add and git commit, and when I git push the branch, this pull request should also be updated. I gave that a shot.

Reply:
Your understanding is correct. Though you need to be careful of what gets checked in when you run `git add`